core/src/main/resources/jenkins/security/csrf/CSRFAdministrativeMonitor/message.jelly

```diff
@@ -22,8 +22,19 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core">
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
     <div class="alert alert-warning">
-        ${%warningMessage(rootURL)}
+        <form method="post" action="${rootURL}/${it.url}/disable">
```

Comment:
Is allowing this monitor to be dismissed important enough for a changelog entry?